### ISSUE TITLE
508 Pre-Launch Accessibility Review for Rated Disabilities

### ISSUE THREAD

**Issue by jason-gcio** [department-of-veterans-affairs/va.gov-team#3418]
## Team Name
eBenefits

## Product Description
The [View Rated Disabilities tool page](https://staging.va.gov/disability/view-disability-rating/rating) displays the Veterans Total Combined Disability Rating, as well as the individual ratings that are used in that calculation, and non-service connected ratings.  Here is the [mockup](https://xd.adobe.com/view/0db723ac-52ff-48b3-4877-9d5882cb2e1f-7b86/screen/90b2e033-e56a-4634-91ff-b694b46d2097/RD-Screen-Tool-FINAL).

## Staging URL and Credentials
- Staging url: https://staging.va.gov/disability/view-disability-rating/rating
- Test-accounts (`usual staging password`)
  - User: vets.gov.user+228@gmail.com (has 5 ratings, 3 are SC)
  - User: vets.gov.user+10@gmail.com (has no ratings)
  - User: vets.gov.user+123@gmail.com (has 1 rating, SC)
  - User: vets.gov.user+38@gmail.com (has 3 rating, 2 are SC)
  - User: vets.gov.user+45@gmail.com (has 1 rating, SC)
  - User: vets.gov.user+232@gmail.com (error)

## Test Plan
The Rated Disabilities page (titled "Your VA disability ratings") has very few moving parts and simply recalls information based on the authentication of the user.  The test plan should include:
- Logging in using several different accounts (see above) and verifying that each looks and behaves as expected
- For those accounts that include errors, the test should validate that those errors look and behave as expected
- All links should have their look and behavior tested for validity

## Known Issues
- Some browser behavior for screen readers were deemed systemic and a ticket was created with Public Websites

**Comment by micahchiang** [comment]
@jason-gcio - Is this ticket still relevant? I ask because I know @jenstrickland is currently doing some accessibility analysis for us.

**Comment by jenstrickland** [comment]
@micahchiang Yes, I am collecting issues from the audit in this epic: https://github.com/department-of-veterans-affairs/va.gov-team/issues/3577 

There is a VoiceOver in Safari issue I was stuck on, and looped in @jason-gcio for some questions. In VO, the Skip to Content link isn't available for about 20 seconds. I'm beginning to suspect this is an issue for VO across the authenticated experience, but am still investigating. Since my Mac is in the shop today, I'll move on to the other screen readers and tests

**Comment by jenstrickland** [comment]
@jason-gcio Are you looking for an accessibility audit? If so, please use one of the issue templates linked here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/review-process.md#development-review

If something else, please let me know. Thanks!

**Comment by jenstrickland** [comment]
Findings will be collected in https://github.com/department-of-veterans-affairs/va.gov-team/issues/4619, and I'll comment here again when the review is complete.

**Comment by jenstrickland** [comment]
@jason-gcio Pre-Launch 508 Review is complete, https://github.com/department-of-veterans-affairs/va.gov-team/issues/4619. There are a few adjustments recommended to improve the experience for those with cognition considerations or using assistive tech.

**Comment by jason-gcio** [comment]
@jenstrickland - Closing this now, but addressing future issues in other tickets.
